### PR TITLE
Declutters Slack case threads

### PR DESCRIPTION
**Pull request by mvilanova**

### CODE REVIEW

File: src/dispatch/plugins/dispatch_slack/case/messages.py [hunk 1]
```diff
@@ -214,7 +211,9 @@ def create_signal_messages(case_id: int, channel_id: str, db_session: Session) -
                 ),
             ]
         ),
-        Section(text=f"Total instances in this case: *{num_of_instances}*\n"),
+        Section(text="*Signal Instances*"),
+        Divider(),
+        Section(text=f"{num_of_instances} signal instances observed in this case."),
```
Comment (wssheldon) [inline src/dispatch/plugins/dispatch_slack/case/messages.py:216]:
```suggestion
        Section(text=f"{num_of_instances} alerts observed in this case."),
```

"Alerts" is more widely understood, in my opinion.

Comment (mvilanova) [reply src/dispatch/plugins/dispatch_slack/case/messages.py]: I already accepted it, but if we use alerts here, should we use it everywhere else too?

File: src/dispatch/plugins/dispatch_slack/case/messages.py [hunk 2]
```diff
@@ -214,7 +211,9 @@ def create_signal_messages(case_id: int, channel_id: str, db_session: Session) -
                 ),
             ]
         ),
-        Section(text=f"Total instances in this case: *{num_of_instances}*\n"),
+        Section(text="*Signal Instances*"),
```
Comment (wssheldon) [inline src/dispatch/plugins/dispatch_slack/case/messages.py:214]:
```suggestion
        Section(text="*Alerts*"),
```

File: src/dispatch/plugins/dispatch_slack/case/messages.py (hunk 1 above)
Comment (wssheldon) [reply src/dispatch/plugins/dispatch_slack/case/messages.py]: In user facing UI, I would say so.

**Pull request merged**